cli/globals/globals.go

```diff
@@ -28,5 +28,6 @@ var (
 	VersionInfo = version.NewInfo(filepath.Base(os.Args[0]))
 	// DefaultIndexURL is the default index url
 	DefaultIndexURL = "https://downloads.arduino.cc/packages/package_index.json"
-	tr              = i18n.Tr
+	// Tr is used to mark strings for translation
```

Comment:
Do not export `globals.Tr`, but define `var tr = i18n.Tr` in the compilation unit where you need a translation.